EFCore.CheckConstraints/CheckConstraints/Internal/EnumCheckConstraintConvention.cs

```diff
@@ -36,9 +41,8 @@ public virtual void ProcessModelFinalizing(IConventionModelBuilder modelBuilder,
 
                         sql.Clear();
 
-                        sql.Append("[");
-                        sql.Append(property.GetColumnName());
-                        sql.Append("] IN ("); ;
+                        sql.Append(_sqlGenerationHelper.DelimitIdentifier(property.GetColumnName()));
```

Comment:
same nullability issue as above.

Reply:
Also changed to follow the logic in https://github.com/efcore/EFCore.CheckConstraints/pull/8#discussion_r487836714.